src/app/app-modules/core/components/allergen-search/allergen-search.component.html

```
@@ -1,99 +1,78 @@
-    matDialogClose
-    matTooltip="{{ current_language_set?.common?.close }}"
-    class="pull-right exit width-20px mat_icon_white"
-    >close</mat-icon
-  >
-</div>
-<div class="container-fluid">
-  <div class="overlay" *ngIf="showProgressBar">
-    <div class="overlay-content">
-      <mat-spinner></mat-spinner>
-    </div>
+<div class="flex flex-col gap-4 p-6">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)